Controller.php

```diff
@@ -40,7 +40,7 @@ protected function getAvailableRanges()
             12 => Piwik::translate('InvalidateReports_XMonths', 12),
             6  => Piwik::translate('InvalidateReports_XMonths', 6),
             3  => Piwik::translate('InvalidateReports_XMonths', 3),
-            1  => Piwik::translate('InvalidateReports_XMonths', 1),
+            1  => Piwik::translate('InvalidateReports_LastMonth', 1),
```

Comment:
You can remove the `1` as parameter in this case, as the translation doesn't have a placeholder

Reply:
You are right, sorry I missed that :)

Btw. very useful plugin! 👍